### DISCUSSION TITLE
Feature request: Add option for hidden arrow taskbar tray to behave like in Vista & XP

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [valinet/ExplorerPatcher#3214] [Ideas]
Hello, I have an idea of adding option for original arrow taskbar tray to behave like expandable toolbar instead of pop-up menu (Like Windows Vista & XP did).

I know, that this wouldn't be practical, but I have one major reason for implementing this, and that's because of consistency (e.g. if someone wants to make his/her Windows 11 looks like Windows Vista or XP).

Here is screenshot:
![Win XP taskbar](https://github.com/valinet/ExplorerPatcher/assets/69351811/329c6e29-e08e-4d68-a0d9-883ce3395688)

NOTE: I'm apologising, that I didn't find suitable screenshot of Windows XP or Vista with good quality and full taskbar view, so I posted an art screenshot instead.

**Comment by pyrates999** [comment]
I think this is beyond what EP is for, sorry.

**Reply by Aleksandra0056** [reply]
I understand. Should I close this and post a feature request on Open-Shell-Menu instead?

**Reply by pyrates999** [reply]
You can try.

**Reply by Aleksandra0056** [reply]
Okay, I posted it into Open-Shell-Menu GitHub, so I'm closing this discussion.

Here is a link if you want: https://github.com/Open-Shell/Open-Shell-Menu/issues/1917
